ctera/src/test/java/ch/cyberduck/core/ctera/CteraMoveFeatureTest.java

```diff
@@ -0,0 +1,161 @@
+        assertFalse(new DAVFindFeature(session).find(folder));
+        assertFalse(new DAVFindFeature(session).find(test));
+        assertTrue(new DAVFindFeature(session).find(target));
+        assertTrue(new DAVFindFeature(session).find(new Path(target, test.getName(), EnumSet.of(Path.Type.file))));
+        assertEquals(attr, new CteraAttributesFinderFeature(session).find(new Path(target, test.getName(), EnumSet.of(Path.Type.file))));
+        assertEquals(attr.getModificationDate(), new CteraAttributesFinderFeature(session).find(new Path(target, test.getName(), EnumSet.of(Path.Type.file))).getModificationDate());
+        assertEquals(attr.getFileId(), new CteraAttributesFinderFeature(session).find(new Path(target, test.getName(), EnumSet.of(Path.Type.file))).getFileId());
+        // TODO CTERA-137 what is the expected behaviour?
+        //assertNotEquals(attr.getETag(), new CteraAttributesFinderFeature(session).find(new Path(target, test.getName(), EnumSet.of(Path.Type.file))).getETag());
+        new CteraDeleteFeature(session).delete(Collections.<Path>singletonList(target), new DisabledLoginCallback(), new Delete.DisabledCallback());
+    }
+
+    @Test
+    public void testMoveOverride() throws Exception {
+        final Path test = new Path(new DefaultHomeFinderService(session).find(), new AlphanumericRandomStringService().random(), EnumSet.of(Path.Type.file));
+        new CteraTouchFeature(session).touch(test, new TransferStatus());
+        final Path target = new Path(new DefaultHomeFinderService(session).find(), new AlphanumericRandomStringService().random(), EnumSet.of(Path.Type.file));
+        new CteraTouchFeature(session).touch(target, new TransferStatus());
+        assertThrows(ConflictException.class, () -> new CteraMoveFeature(session).move(test, target, new TransferStatus().exists(false), new Delete.DisabledCallback(), new DisabledConnectionCallback()));
+        new CteraMoveFeature(session).move(test, target, new TransferStatus().exists(true), new Delete.DisabledCallback(), new DisabledConnectionCallback());
+        assertFalse(new DAVFindFeature(session).find(test));
+        assertTrue(new DAVFindFeature(session).find(target));
+        new CteraDeleteFeature(session).delete(Collections.<Path>singletonList(target), new DisabledLoginCallback(), new Delete.DisabledCallback());
+    }
+
+    @Test(expected = NotfoundException.class)
+    public void testMoveNotFound() throws Exception {
+        final Path test = new Path(new DefaultHomeFinderService(session).find(), new AlphanumericRandomStringService().random(), EnumSet.of(Path.Type.file));
+        new CteraMoveFeature(session).move(test, new Path(new DefaultHomeFinderService(session).find(), new AlphanumericRandomStringService().random(), EnumSet.of(Path.Type.file)), new TransferStatus(), new Delete.DisabledCallback(), new DisabledConnectionCallback());
+    }
+
+    @Test
+    public void testPreflightNoCustomProps() throws BackgroundException {
```

Comment:
@dkocher does not make sense to implement these 4 `preflight` tests for all features according to list in `CteraSession`?

Reply:
offline discussion with @dkocher : yes.